src/components/index/createRequestsTableCells.tsx

```diff
@@ -72,18 +73,19 @@ function createRequestsTableCells(
         ? req.proposedPrice.toString()
         : undefined;
 
-      if (proposedPrice && unanswerable.includes(proposedPrice)) {
+      // req.state 1 is fresh request with no proposed answer, in that case always set to "-"
+      // proposed price sometimes defaults to 0 even if noone has proposed an answer yet.
+      if (req.state === undefined || req.state <= 1) {
+        proposedPrice = "-";
+      } else if (proposedPrice && unanswerable.includes(proposedPrice)) {
         proposedPrice = "Requested too early";
       } else if (
         proposedPrice &&
         proposedPrice !== "0" &&
         identifier !== "YES_OR_NO_QUERY"
```

Comment:
this was a bug, identifier was deceptively defined, it was actually the title of the request. This has been fixed above, there is now a title variable and the correct identifier value